src/CMakeLists.txt

```diff
@@ -68,14 +67,27 @@ find_package(OpenMP QUIET)
 find_package(ZLIB   QUIET)
 find_package(BZip2  QUIET)
 find_package(SeqAn  QUIET REQUIRED CONFIG)
-find_package(SeqAn3 QUIET REQUIRED CONFIG)
+
+# BIO
+find_package (biocpp COMPONENTS core io QUIET REQUIRED HINTS "${CMAKE_SOURCE_DIR}/submodules/biocpp-core/build_system")
+
+# SHARG
+find_package (sharg QUIET REQUIRED HINTS "${CMAKE_SOURCE_DIR}/submodules/sharg-parser/build_system")
+
+# CEREAL
+if (NOT EXISTS ${CMAKE_SOURCE_DIR}/submodules/cereal)
+    find_package(cereal QUIET REQUIRED)
+endif ()
 
 message(STATUS "These dependencies were found:")
-message(   "     OPENMP     ${OPENMP_FOUND}      ${OpenMP_CXX_FLAGS}")
-message(   "     ZLIB       ${ZLIB_FOUND}      ${ZLIB_VERSION_STRING}")
-message(   "     BZIP2      ${BZIP2_FOUND}      ${BZIP2_VERSION_STRING}")
-message(   "     SEQAN      ${SEQAN_FOUND}      ${SEQAN_VERSION_STRING}")
-message(   "     SEQAN3     ${SEQAN3_FOUND}      ${SEQAN3_VERSION}")
+message(   "     BIOCPP-CORE  ${BIOCPP_CORE_FOUND}      ${BIOCPP_CORE_VERSION}")
```

Comment:
I receive the following message when running cmake, do you know why some variables are missing?

```
-- These dependencies were found:
     BIOCPP-CORE  TRUE      
     BIOCPP-IO    TRUE      
     BZIP2        TRUE      1.0.8
     CEREAL             
     OPENMP       TRUE      -fopenmp
     SEQAN        TRUE      2.4.1
     SHARG        TRUE      1.0.0
     ZLIB         TRUE      1.2.12
```

Reply:
Yes, the versions missing from BIOCPP are due to bug that I already fixed upstream: biocpp/biocpp-core#57
(but we are tracking the release version of biocpp-core, so we will get it later.

Cereal's stuff missing is due to the fact, that you cannot use `find_package()` when using cereal from a git-submodule (at least I haven't figured out a way). So it's nothing we can change. [If the cereal submodule is not cloned with the source code, lambda will look for a system-wide cereal install, and then the version will also appear]

Reply:
thanks!